commands/previewPush.go

```diff
@@ -161,10 +161,13 @@ DomainLoop:
 		fmt.Fprintf(os.Stderr, "##teamcity[buildStatus status='SUCCESS' text='%d corrections']", totalCorrections)
 	}
 	notifier.Done()
-	out.Printf("Done. %d corrections.\n", totalCorrections)
```

Comment:
I feel like we need to still print the same thing in the 0 corrections case. There may be automations depending on that text. They can switch to status code now, but here we are.

I don't mind if the output looks a little redundant, or you could add a little more logic to make it sane. But I do fear something will be looking for that exact output.

Reply:
I'll change it back.

Reply:
Fixed.